docs/pages/enterprise/workflow/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Access Requests for SSH and Kubernetes Access
-description: How to customize SSH and Kubernetes access using Teleport.
+title: Access Requests for SSH, Kubernetes, Databases, Desktops and Web Apps.
+description: Teleport allows users to request elevated privileges in the middle of their command-line sessions.  Requests can be escalted via ChatOps or anywhere else via flexible Authorization Workflow API.
```

Comment:
You added desktops, but then only mention "command-line" sessions in the description, which isn't something you can do with desktop access.

Reply:
I made it more generic to apply to roles via the CLI or UI.